docs/source/qualifiers.md

```diff
@@ -0,0 +1,96 @@
+# Definition of qualifier classes used in `clinlp`
+
+This page describes the definitions of qualifier classes for Dutch clinical text we use in `clinlp`.
+
+## Introduction
+
+When processing clinical documents (e.g., doctor’s notes, discharge letters), detecting qualifiers (e.g., `absent`, `historical`, `non-patient`, `increasing`, `decreasing`, `severe`, `light`, etc.) follows the matching of concepts (e.g., diagnoses, symptoms, procedures). In `clinlp` we primarily use the term “qualifier”, although the terms “context”, “meta-annotation”, and “modifier” have also been used to denote the same concept.
+
+A consensus on potential qualifier classes, along with **clear** definitions of the qualifiers they encompass, is needed to develop accurate algorithms for detecting them. Despite some shared intuitive understanding of recognizing, for instance, a _negation_ in a sentence, there are numerous cases where intuition simply falls short. In practice this has impeded manual annotation of gold standards, used for training and evaluating algorithms. In turn the resulting annotations (with Kappa-Cohen as low as 0.5) lead to a difficult target for supervised machine learning models. A standardized classification of qualifiers, as proposed here, will hopefully advance both research and clinical implementation of NLP (Natural Language Processing) algorithms. This page is the result of some deliberation among various researchers and developers working with clinical text, but it is not necessarily definitive. We welcome feedback and suggestions for improvement.
+
+For these definitions we took the Context Algorithm (Harkema, 2009) as a starting point, both because this is an influential paper, and because a corresponding Dutch corpus is available (Afzal, 2014). There are already some trained models available that can for a large part be re-used. We will here describe three qualifier classes: **Presence**, **Temporality**, and **Experiencer**, including some definitions, issues to resolve, and illustrative examples. These classes can be further de-aggregated at a later stage, and other classes may follow later as well. Note that choosing qualifiers is a trade-off between granularity and practicality. We aim for a balance that is useful for most clinical NLP tasks.
```

Comment:
Nog zinnig om hier url's aan de referenties toe te voegen? Lijkt me wel in lijn met de rest van de documentatie.